x/ibc/04-channel/types/msgs_test.go

```diff
@@ -347,3 +351,130 @@ func (suite *MsgTestSuite) TestMsgChannelCloseConfirm() {
 		}
 	}
 }
+
+var _ exported.PacketDataI = validPacketT{}
+
+type validPacketT struct{}
```

Comment:
please don't use mocked types. Might lead to unchecked errors. Use the `Packet` type instead, as that's what will be used on the messages

Reply:
@cwgoes you can create a valid `types.Packet` instead of creating a custom type that implements the interface

Reply:
`types.Packet` has a `PacketDataI` member, that's what this is being used for